test/unit/player.test.js

```diff
@@ -67,7 +67,7 @@ test('should accept options from multiple sources and override in correct order'
   videojs.options.attr = 1;
 
   let tag0 = TestHelpers.makeTag();
-  let player0 = new Player(tag0);
+  let player0 = new Player(tag0, { techOrder: ['html5'] });
```

Comment:
We have a 'techFaker' tech that I think could be used here instead of html5. It would be technically safer on IE8.